Source/Core/UICommon/GameFile.cpp

```diff
@@ -149,13 +153,83 @@ bool GameFile::IsValid() const
   return true;
 }
 
+bool GameFile::UpdateCover()
+{
+  if (!m_cover.buffer.empty() || !Config::Get(Config::MAIN_USE_GAME_COVERS))
+    return false;
+
+  Common::HttpRequest request;
+
+  std::string region_code;
+
+  const bool is_wii = GetPlatform() != DiscIO::Platform::GameCubeDisc;
+
+  auto user_lang = SConfig::GetInstance().GetCurrentLanguage(is_wii);
```

Comment:
This makes Dolphin crash on Android (unfortunately...). I'd suggest putting an Android ifdef at the top of this function that makes us return false, since this functionality isn't used on Android anyway.

Reply:
Alternatively, you could break this code out to a new function and call it here: https://github.com/spycrab/dolphin/blob/0010bb6c8df9941610925e7b5c5f93381bb8f7cd/Source/Core/UICommon/GameFile.cpp#L86-L93